library/src/main/res/values/text_appearances.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You can remove this file and use `TextAppearance.AppCompat.Body1` in support libray